test/SubPoolFactory.t.sol

```diff
@@ -9,18 +9,18 @@ import {ERC20} from "solady/tokens/ERC20.sol";
 contract SubPoolFactoryTest is BaseTest {
     MockToken lst = new MockToken("LST", "ether");
     address solver = makeAddr("solver");
-    address solverPoolAddress;
-    uint256 solverEthAmt = 20 ether;
-    uint256 solverCowAmt = 100_000 ether;
+    address payable solverPoolAddress;
+    uint256 solverTokenAmt = 20 ether;
     SubPool solverPool;
     address notOwner = makeAddr("notOwner");
     string backendUri = "https://backend.solver.com";
 
     function setUp() public override {
         super.setUp();
 
-        solverPoolAddress = _seedAndDeployPool(solver, TOKEN_WETH_MAINNET, solverEthAmt, solverCowAmt, 0, backendUri);
-        solverPool = SubPool(payable(solverPoolAddress));
+        solverPoolAddress =
+            _seedAndDeployPool(solver, TOKEN_WETH_MAINNET, solverTokenAmt, solverTokenAmt, 0, backendUri);
+        solverPool = SubPool(solverPoolAddress);
```

Comment:
I see this change comes from [this suggestion](https://github.com/meetmangukiya/cow-decettle/pull/1/#discussion_r1700379757), which unfortunately was written in a misleading way.
It's very reasonable to use two different amounts `solverEthAmt` and `solverCowAmt` since they do different things. The nit was about using in the specific test the constant value `15 ether` for the ETH amount and a variable value `solverCowAmt` for the COW amount without there being a meaningful difference between the two in the test. So it was just a suggestion to define `uint256 solverEthAmt = 15 ether;` in that test or reuse `solverEthAmt`.

Reply:
ahh i would just leave this as it is now, i don't think it is materially changing anything to the tests which is why i made the nit change, dont want to go through the renaming back